README.md

```diff
@@ -343,9 +343,30 @@ foreach ($annotation->faces() as $key => $face) {
 }
 ```
 
+## Caching Access Tokens
+
+By default the library will use a simple in-memory caching implementation, however it is possible to override this behavior by passing a [PSR-6](http://www.php-fig.org/psr/psr-6/) caching implementation in to the desired client.
+
+The following example takes advantage of [Symfony's Cache Component](https://github.com/symfony/cache).
+
+```php
+<?php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Storage\StorageClient;
+use Symfony\Component\Cache\Adapter\ArrayAdapter;
+
+// Choose the implementation that fits your needs.
+$cache = new ArrayAdapter();
```

Comment:
I would mention about the security implication of the cache with a proper example using FileSystem cache.

Example:

```php
$cache = new Stash\Driver\FileSystem();
// Set strict permission to protect the access token.
$options = array('filePermissions' => 0600);
$cache->setOptions($options);
```

or something?